Scripts/Runtime/Logging/.CSProject/Logging_StaticStateResetHelper.cs.meta

```diff
@@ -1,3 +0,0 @@
-fileFormatVersion: 2
-guid: 87c32da738604588800033dca9be761f
-timeCreated: 1657901177
```

Comment:
I can restore it if necessary but - did this need a .meta file? It's within the .CSProject so Unity doesn't see it right?

Reply:
You're right. Should probably ignore all meta files under the .CSProject directory to avoid this accident in the future. (new PR)

Reply:
I mean, normally that's not a concern - Unity can't see this folder to generate the .meta files in the first place. Maybe this file started in Unity and was moved up? Either way, harmless.

Reply:
Yea or Rider auto created it.